collectd/files/backend/carbon.conf

```diff
@@ -1,7 +1,8 @@
-{%- from "linux/map.jinja" import system with context %}
 {%- set backend = salt['pillar.get']('collectd:client:backend:'+backend_name) %}
-
-LoadPlugin write_graphite
+{%- from "linux/map.jinja" import system with context %}
```

Comment:
Maybe it's time for getting rid of this weirdness and sick dependency on linux:
```
Prefix "{{ system.get('cluster', 'prd') }}_{{ system.get('environment', 'prd') }}."
```
and make it configurable as a part of collectd metadata.